Doc/howto/enum.rst

```diff
@@ -1123,6 +1143,21 @@ The most interesting thing about enum members is that they are singletons.
 and then puts a custom :meth:`__new__` in place to ensure that no new ones are
 ever instantiated by returning only the existing member instances.
 
+Flag Members
+^^^^^^^^^^^^
+
+Flag members can be iterated over just like the Flag class, and only the
+canonical members will be returned.  For example,
+``list(Color) ==  [<Color.RED: 1>, <Color.GREEN: 2>, <Color.BLUE: 4>]``.
+(Note that ``BLACK``, ``PURPLE``, and ``WHITE`` do not show up.)
+
+Inverting a flag member does not return a negative value, but the
+corresponding positive value -- for example,
+``~Color.RED == <Color.GREEN|BLUE: 6>``.
+
+Flag members have a length corresponding to the number of power-of-two values
+they contain.  For example, ``len(Color.PURPLE) == 2``.
+
 
 .. _enum-cookbook:
```

Comment:
Seems like you're missing a top-level section heading here? This paragraph and ref target appears to be refering to a new set of distinct topic, but is under the `How are Enums and Flags different` heading, and this intro paragraph is nunder the `Flag Members` section. Presumably, this is what you intend, and it got lost at some point:

```suggestion
.. _enum-cookbook:

Enum Cookbook
-------------
```